api/src/test/java/org/openmrs/validator/VisitValidatorTest.java

```diff
@@ -441,11 +441,44 @@ public void validate_shouldPassValidationIfFieldLengthsAreCorrect() {
 	@Test
 	public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() {
 		Visit visit = makeVisit(42);
-		visit
-		        .setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
-		
+		visit.setVoidReason("too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text too long text");
```

Comment:
@dkayiwa Yes I did. It looked somewhat bizarre to me, so I just decided to put it on one line. Does it pose a problem?